kitsune/settings.py

```diff
@@ -757,6 +758,16 @@ def JINJA_CONFIG():
 # Default storage engine - ours does not preserve filenames
 DEFAULT_FILE_STORAGE = 'kitsune.upload.storage.RenameFileStorage'
 
+# AWS S3 Storage Settings
+# local_settings must include the following to use s3 storage
+# AWS_ACCESS_KEY_ID
+# AWS_SECRET_ACCESS_KEY
+# AWS_STORAGE_BUCKET_NAME
+AWS_S3_OBJECT_PARAMETERS = {
+    'CacheControl': 'max-age=86400',
```

Comment:
Current max age is `2592000`

Reply:
Please keep this the same value unless there's a reason to change it.